test/regress/cli/CMakeLists.txt

```diff
@@ -3823,6 +3835,7 @@ set(regress_2_tests
   regress2/strings/repl-repl.smt2
   regress2/strings/replace-re-all-unsat.smt2
   regress2/strings/replace_re.smt2
+  regress2/strings/replace-re-all-unsat.smt2
```

Comment:
@yoni206 Its this one, apparently the git merging has been a bit error prone.  Good catch.